ci/jenkins/jobs.yaml

```diff
@@ -34,20 +34,20 @@
         - openstack
         - vmware
     test:
-        - 'test_hard_reboot'
+        - 'test_node_reboot'
         - 'test_upgrade_plan_all_fine'
         - 'test_upgrade_plan_from_previous'
         - 'test_upgrade_plan_from_previous_with_upgraded_control_plane'
         - 'test_upgrade_apply_all_fine'
         - 'test_upgrade_apply_from_previous'
         - 'test_upgrade_apply_user_lock'
         - 'test_cilium'
-        - 'test_deployments'
```

Comment:
Curious: why make this change? The test is named test_nginx_deployment, but if I understand correctly, the suite will be executed, so the name of the suite can be still the same.

Reply:
just to make clear what it contains I guess? The suite being called `deployments` makes me feel like it tests several things while its just testing deploying nginx.

Could probably be more generic if we expand the test to deploy other types of items to test different types of deployments (sidecards in the future? pvcs?)